pytometry/plotting/_scatter_density.py

```diff
@@ -47,12 +47,10 @@ def scatter_density(
         ax (`matplotlib.Axes`), optional:
             Axes to draw into. If *None*, create a new figure or use ``fignum`` to
             draw into an existing figure.
-        cmap (str or list or :class:`matplotlib.cm.Colormap`), optional:
+        cmap (str or list or :class:`matplotlib.colors.Colormap`), optional:
             For scalar aggregates, a matplotlib colormap name or instance.
             Alternatively, an iterable of colors can be passed and will be converted
-            to a colormap. For a single-color, transparency-based colormap, see
-            :func:`alpha_colormap`.
-            Defaults to 'jet'.
```

Comment:
There's no `alpha_colormap` in pytometry, nor could I find that function in matplotlib.